File: PRAKTIKUM 4/PRAK403_2310817220028_SheilaSabina.py
satu, dua = map(int, input("Input\n").split())

print("\nOutput")
if satu < dua:
    for s in range(satu, dua+1):
        print(s, satu+dua-s, end="-" if s < dua else "")
elif satu > dua:
    for s in range(satu, dua-1, -1):
        print(s, satu+dua-s, end="-" if s > dua else "")
else:
    print(satu, dua)

File: PRAKTIKUM 4/PRAK403_2310817220028_SheilaSabina.c
#include <stdio.h>
int main () {
    int satu, dua;

    printf("Input\n");
    scanf("%d %d", &satu, &dua);
    
    printf("\nOutput\n");
    if(satu < dua){
        for (int i = satu; i <= dua; i++){
            printf("%d %d", i, satu + dua - i);
            if (i < dua){
                printf("-");
            }
        }
    } else if (satu > dua){
        for (int i = satu; i >= dua; i--){
            printf("%d %d", i, satu + dua - i);
            if (i > dua){
                printf("-");
            }
        }
    } else {
        printf("%d %d", satu, dua);
    }
    return 0;
}